### PR TITLE
 Import 'io' instead of 'StringIO' on Python 3 #295

### PR DESCRIPTION
**Pull request by Abhishek-Dev09**
A common idiom in Python 2 was to try to import cStringIO as StringIO. If that failed, it tried to import import StringIO instead. Do not do this in Python 3; the io module does it for you. It will find the fastest implementation available and use it automatically.

Signed-off-by: Abhishek Kumar abhishek.kasyap09@gmail.com

### CODE REVIEW

**Comment by Abhishek-Dev09** [comment]
@pombredanne : Can i change here also https://github.com/nexB/scancode-toolkit/blob/develop/tests/scancode/data/multiprocessing/patchelf.pdf , https://github.com/nexB/scancode-toolkit/blob/develop/tests/textcode/data/pdf/pdf.pdf , https://github.com/nexB/scancode-toolkit/blob/develop/tests/typecode/data/contenttype/doc/pdf/pdf.pdf . I think i am getting issue due to these files.

**Comment by pombredanne** [comment]
@Abhishek-Dev09  re:  

> Can i change here also https://github.com/nexB/scancode-toolkit/blob/develop/tests/scancode/data/multiprocessing/patchelf.pdf , https://github.com/nexB/scancode-toolkit/blob/develop/tests/textcode/data/pdf/pdf.pdf , https://github.com/nexB/scancode-toolkit/blob/develop/tests/typecode/data/contenttype/doc/pdf/pdf.pdf . I think i am getting issue due to these files.

Absolutely not: these are test files that are there on purpose. Not being able to handle them means that code needs to fixed... and not the tests changed. (Which is a general principle BTW)

**Comment by Abhishek-Dev09** [comment]
@pombredanne : I got the same error that you got in travis of https://github.com/nexB/scancode-toolkit/pull/1609 . Can i you got it where i am wrong?

File: tests/textcode/test_pdf.py [hunk 1]
```diff
@@ -44,7 +44,7 @@ def test_get_text_lines(self):
 Extracts text from a pdf file.
 """
 import contextlib
-from StringIO import StringIO
+from io import StringIO
```
Comment (pombredanne) [inline tests/textcode/test_pdf.py]: DO NOT change this code... this is  an expectation test string. This is not code... even if it looks like it. See line 41   https://github.com/nexB/scancode-toolkit/pull/1608/files#diff-4fbe57782889037fd619e1df7509d369R41 ... this is a string!

Comment (Abhishek-Dev09) [reply tests/textcode/test_pdf.py]: oh, but you have done here https://github.com/nexB/scancode-toolkit/commit/1e95b2c025acc82aec3e673012a04ee96f808602#diff-4fbe57782889037fd619e1df7509d369 ,  the test is still passed.

Comment (Abhishek-Dev09) [reply tests/textcode/test_pdf.py]: Ok, I removing this, but i want to know why your commits passed all test.

Comment (pombredanne) [reply tests/textcode/test_pdf.py]: @Abhishek-Dev09 you want not to look at a single commit at a time but at the end game with all commits applied.

Comment (pombredanne) [reply tests/textcode/test_pdf.py]: And that https://github.com/nexB/scancode-toolkit/commit/1e95b2c025acc82aec3e673012a04ee96f808602#diff-4fbe57782889037fd619e1df7509d369L47 surely was a mistake of mine ;)

**Comment by pombredanne** [comment]
@Abhishek-Dev09 this need rebasing now.

**Comment by pombredanne** [comment]
thanks!

**Comment by pombredanne** [comment]
All set. Thanks! merging

**Pull request merged**